js/components/accessibilityControls.js

```diff
@@ -3,33 +3,45 @@ var Utils = require('./utils');
 
 var AccessibilityControls = function (controller) {
   this.controller = controller;
+  this.allowed = true;
```

Comment:
Can we use a more descriptive name for this var? From the name it's not clear what exactly is allowed or not.

Reply:
\+